coalib/bearlib/abstractions/Linter.py

```diff
@@ -43,6 +44,15 @@ def _prepare_options(options, bear_class):
     if not options['use_stdout'] and not options['use_stderr']:
         raise ValueError('No output streams provided at all.')
 
+    if options['convert_index']:
```

Comment:
dont abuse strings like this.

there are four possible scenarios, being the combinations of.

- increment line numbers, or dont touch them. 
- increment column numbers, or dont touch them.

those should be two meaningful arg names, `bool` obviously.

arg names are checked by the Python runtime, providing consistent error messages.  the user documentation becomes easy.  etc. etc.